### PR TITLE
[ja] follows #5574, #5575

### PR DESCRIPTION
**Pull request by okinaka**

### CODE REVIEW

File: ja/views/cells.rst [hunk 1]
```diff
@@ -249,3 +253,35 @@
 
 .. versionadded:: 3.5.0
     ``Cake\Datasource\Paginator`` は 3.5.0 で追加されました。
+
+セルのオプション
+=================
+
+セルは、セルオブジェクトの作成時にプロパティに変換されるコンストラクターオプションを宣言できます。 ::
```
Comment (chinpei215) [inline ja/views/cells.rst]: プロパティー

**Comment by okinaka** [comment]
@chinpei215 Thanks! I updated.

File: ja/views/cells.rst [hunk 2]
```diff
@@ -249,3 +253,35 @@
...
+
+        public function display($userId)
+        {
+            $this->loadModel('Users');
+            $result = $this->Users->find('friends', ['for' => $userId]);
+            $this->set('favorites', $result);
+        }
+    }
+
+ここでは、 ``$limit`` プロパティを定義し、 ``limit`` をセルのオプションとして追加しました。
```
Comment (chinpei215) [inline ja/views/cells.rst]: プロパティー

File: ja/views/cells.rst [hunk 3]
```diff
@@ -249,3 +253,35 @@
...
+これにより、セルの作成時にオプションを定義することができます。 ::
+
+    $cell = $this->cell('Favorites', [$user->id], ['limit' => 10])
+
+セルのオプションは、データをプロパティとして使用してデフォルト値を
```
Comment (chinpei215) [inline ja/views/cells.rst]: プロパティー

Sorry. I should have pointed out all of them.

**Comment by okinaka** [comment]
@chinpei215 Thanks! I updated.

**Comment by chinpei215** [comment]
@okinaka Thank you.

**Pull request merged**